lib/gql_serializer.rb

```diff
@@ -27,11 +27,11 @@ def self.parse_query(input)
 
   def self.query_include(model, hasharray)
     include_array = []
-    relations = model.reflections.keys
+    relations = model.reflections
     hasharray.each do |e|
       if e.is_a? String
         key = e.split(':')[0]
-        include_array.push(key) if relations.include?(key)
+        include_array.push(key) if relations[key]
```

Comment:
Can you talk about why you took this approach instead?

Reply:
Sure. What I was doing before was getting reflections, which is a hash, putting the keys into an array, and then searching the array instead of just leaving it as a hash and checking if there's a value for the key I want. It was just wasteful.

example
```ruby
#before
relations = {tags: <TagRelation>, panels: <PanelRelation>}.keys # ["tags", "panels"]
relations.include?("tags")
#after
relations = {tags: <TagRelation>, panels: <PanelRelation>}
!!relations["tags"]
```

Reply:
Thanks!